### PR TITLE
Create umip-32.md

### PR DESCRIPTION
**Pull request by volmexcole**

### CODE REVIEW

File: UMIPs/umip-32.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The DVM should support requests for a price that resolves to the ETHV Index v1 value. 
+
+## Motivation
+The DVM currently does not support ETHV Index v1 or any other volatility indices. Volatility indices / products are a staple of the traiditional financial system, allowing market participants to speculate / hedge volatility. No on-chain volatility products exist today. Adding ETHV Index v1 would enable the first on-chain volatility products.  
+
+## Technical Specification
+
+The definition of this identifier should be:
+- Identifier name: ETHV Index v1
+- Base Currency: USD
```
Comment (chicfilabae) [inline UMIPs/umip-32.md:21]: Interesting- what's the rationale behind keeping the base as USD here?

File: UMIPs/umip-32.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+| Status     | Draft                                                                                                                                         |
+| Created    | December 22, 2020                                                                                                                              
+
+## Summary 
+This UMIP will reference a synthetic token to be created with this price identifier. This token will be referred to as 'ETHV' and will represent the token that tracks this identifier.
+
+The DVM should support requests for a price that resolves to the ETHV Index v1 value. 
+
+## Motivation
+The DVM currently does not support ETHV Index v1 or any other volatility indices. Volatility indices / products are a staple of the traiditional financial system, allowing market participants to speculate / hedge volatility. No on-chain volatility products exist today. Adding ETHV Index v1 would enable the first on-chain volatility products.
```
Comment (chicfilabae) [inline UMIPs/umip-32.md:15]: OOC, what research have you done that gives insight to demand component for this sort of on-chain vol idx? How would you compare this to holding short-dated straddles?

**Review by chicfilabae — CHANGES_REQUESTED** [review]
Left some thoughts here.

**Comment by chicfilabae** [comment]
Also, keep in mind uLABs has spec'd out a BTC vol idx as well

**Comment by volmexcole** [comment]
Will submit an amended proposal. Thanks for the thoughtful feedback

**Comment by chicfilabae** [comment]
@volmexcole In the future don't close the PR + re-open a new one, just add additional commits to the branch so this PR gets updated- best to keep all feedback in a centralized place for others, especially voters who can see the full seqeuence of feedback without having to stitch it together from multiple PRs

**Pull request closed without merging**